enacts/calc.py

```diff
@@ -82,7 +80,7 @@ def read_enacts(variable, ds_conf="dekadal", as_array=True):
             data_path = ds_conf['vars'][variable][0]
         xrds = xr.open_zarr(f"{ds_conf['zarr_path']}{data_path}")
         array = ds_conf['vars'][variable][2]
```

Comment:
`array` is an unhelpful name here. I assumed it was a DataArray, but then saw it being used as `xrds[array]`, and had to go look at the config to confirm that it's actually the name of the variable in the zarr file. Something like `zarr_varname` might be better; alternatively, change the zarrification script to always put the application's standard names (e.g. `tmax` and `tmin` instead of `temp`) in the zarrs, so we don't even need to look at `var_config[2]` when reading.